data/types/bolus/biphasic/biphasic.go

```diff
@@ -0,0 +1,75 @@
+package biphasic
+
+import (
+	"github.com/tidepool-org/platform/data"
+	"github.com/tidepool-org/platform/data/types/bolus/normal"
+	"github.com/tidepool-org/platform/structure"
+)
+
+const (
+	SubType = normal.BiphasicSubType
```

Comment:
Why it is not a "normal" subType, and put things in the normal bolus ?
I would either have:
- move biphasic code under the normal type, remove the biphasic type ? add optional fields to the normal bolus for the biphasic.
- directly add "normal"  / "expectedNormal" to the biphasic

I'm not fan of having subType melt this way. I think we should keep them separated.

Reply:
We could have everything in the bolus object as well ;-) 
The idea was to have a separate bolus (sub)type to be ease their identification. 
- Bringing back the normal/expectedNormal fields in Biphasic would just duplicate the code. 
- Merging Normal and Biphasic would make the code more complex.